engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.content-hosts.factory:Host
```

Comment:
Also, the test needs to be renamed to from `hosts.factory.js` to `host.factory.js`